IPs/RSKIP194.md

```diff
@@ -0,0 +1,108 @@
+```
+
+The *mergeMiningFields* are the following:
+
+```
+bitcoinMergedMiningHeader,
+bitcoinMergedMiningMerkleProof,
+bitcoinMergedMiningCoinbaseTransaction
+```
+
+All blocks prior the hard-fork block number are considered version 0.
+Afterward, new blocks must be version 1.
+For block headers version 1, there are two separate serialization methods, one for the transfer of the block header information and the other for the computation of the block hash.
+When serializing the block header version 1 to obtain the block hash, the field logsBloom is replaced by an RLP list of:
+
+* The blockVersion field (integer) (1 byte)
+* The Keccak hash of the bloom filter data (32 bytes).
+
+When serializing the block header for transmission or storage, the logsBloom field is: 
+* The blockVersion field (integer) (1 byte)
+* The bloom filter data (256 bytes).
+
+Consensus must validate that the blockVersion is exactly 1 for blocks after the hard fork, until this number is incremented by another consensus change.
+
+
+## Rationale
+
+Generally the blockVersion is stored as the first field of an object.
+While this is entirely possible for RSK block headers, PowHSMs cannot understand any other block header format. However, PowHSM can verify without problem a block header with shorter data stored in the logsBloom field.
+To avoid forcing a costly firmware upgrade for all PowHSMs, the blockVersion is stored in the logsBloom filter, which is unchecked by the hardware.
+
+While computing the hash without the full bloom filter data may incentivize nodes to remove this data permanently, the overhead of the bloom filter data is very low for blocks with many transaction. Therefore this incentive is weak. 
+
+A improved version of this RSKIP would also hash the fields coinbase, stateRoot, txTrieRoot, receiptTrieRoot, gasLimit, gasUsed, extraData, paidFees, minimumGasPrice, uncleCount and ummRoot into the same inner hash, saving another 145 bytes. This change converts the RSK header into a mini-header, resembling the old [mini-blockchain](https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.1046.128&rep=rep1&type=pdf) proposal (updated [here](http://cryptochainuni.com/wp-content/uploads/The-Mini-Blockchain-Scheme.pdf)). However this change requires the upgrade of the PowHSM firmware, and therefore it should be considered if there is a PowHSM firmware upgrade agreed by the community. This technique can be later combined by [coinprune](https://arxiv.org/pdf/2004.06911.pdf)-like method to reduce the blockchain size even more.
+
+## Backwards Compatibility
+
+This change is a hard-fork and therefore all full nodes must be updated. 
+
+Block explorers do not need to be updated, as the RPC interface will return the serialization block header that corresponds with the expanded version of the block header.
+
+SPV light-clients won't be able to use the all the benefits of this change until they can request from peers the compressed format of block headers.
```

Comment:
Minor typo: remove the first "the" from "won't be able to use the all the benefits"